assets/app/scripts/filters/resources.js

```diff
@@ -58,6 +46,22 @@ angular.module('openshiftConsole')
       return null;
     };
   })
+  .filter('tagAnnotation', function() {
```

Comment:
add a comment that this is for looking up annotations on ImageStream.spec.tags[tag].annotations